src/main/MainTabsScreen.js

```diff
@@ -67,14 +66,7 @@ export default function MainTabsScreen(props: Props) {
 
   return (
     <View style={[styles.flexed, { backgroundColor }]}>
-      {orientation === 'PORTRAIT' && (
-        <View
-          style={{
-            height: insets.top,
-            backgroundColor: DEFAULT_TITLE_BACKGROUND_COLOR,
```

Comment:
The value of this constant is `'transparent'`, which is the default
    background color of `View`s anyway.

It looks like after this change a further simplification is possible: I believe it's the case that `DEFAULT_TITLE_BACKGROUND_COLOR` is only ever used as a sentinel value! That is, it signifies to our own code "there's nothing here, just use your default behavior", and we have conditionals that test for equality to it... but I believe we never pass it as an actual style prop, or otherwise to any code outside our own.

If so, that means that we could replace the constant's initializer with `undefined`, or for that matter `{}` or anything that makes a fresh unique object. And probably better -- we could just replace every use of it with `undefined`, and augment the relative types with `| void` to make that possibility explicit.

Reply:
Hmm, interesting! I think I see just one place where we pass it to code that's not ours, at this commit:

src/chat/ChatScreen
```js
  const titleBackgroundColor = useSelector(state => getTitleBackgroundColor(state, narrow));
  // …
  return (
    <ActionSheetProvider>
      <View style={[componentStyles.screen, { backgroundColor }]}>
        <KeyboardAvoider style={styles.flexed} behavior="padding">
          {orientation === 'PORTRAIT' && (
            <View
              style={{
                height: insets.top,
                backgroundColor: titleBackgroundColor,
              }}
            />
          )}
```

`getTitleBackgroundColor` will return `DEFAULT_TITLE_BACKGROUND_COLOR` if the narrow is not a stream or topic narrow.

Reply:
Ah, right—but passing `undefined` would be just the same as passing 'transparent' there.

Reply:
My new revision makes `DEFAULT_TITLE_BACKGROUND_COLOR` go away. I've also renamed `getTitleBackgroundColor` to `getStreamColorForNarrow`, but I've kept the part of its interface that says it accepts narrows other than stream or topic narrows (for which it returns `undefined`). The `react-hooks/rules-of-hooks` rule doesn't like it if I conditionally use a hook, as in

```
const backgroundColor = isStreamNarrow(narrow)
    ? useSelector(state => getStreamColorForNarrow(state, narrow))
    : '…';
```

```
React Hook "useSelector" is called conditionally. React Hooks must be called in the exact same order in every component (render.eslintreact-hooks/rules-of-hooks)
```